packages/core/core/src/services/entity-validator/validators.ts

```diff
@@ -1,3 +1,13 @@
+/**
+ * Validators check if the entry data meets specific criteria before saving or publishing.
+ * (e.g., length, range, format).
+ *
+ * Drafts have limited validations (mainly max constraints),
```

Comment:
I guess this is more of a product question just for my curiosity, but do you know why we decided to only check max for drafts?

Reply:
I believe the idea is for the user to save an "unfinished" version of an entry, that means being able to save with everything empty and with the minimum friction. Max validations should not be an intrusive validation while drafting an entry, compared to the rest that is